### DISCUSSION TITLE
Create object within object

### DISCUSSION THREAD

**Discussion by JungeWerther** [twentyhq/twenty#3367] [Ideas]
For example, when I add a new person in the People table, when adding a Company field, if no company exists, pressing ENTER should add a new company with the same name, and take me to the relevant card.

**Comment by Bonapara** [comment]
Hi @JungeWerther, sorry for the delayed response! Thank you for the great suggestion. We should implement it within the next six months.

**Comment by BOHEUS** [comment]
Now it's possible to add new record while updating related record 
<img width="1680" height="1014" alt="image" src="https://github.com/user-attachments/assets/26660ade-dc1e-42fc-a699-9bae2a739e0b" />
